src/foam/core/console/cmd/Commands.js

```diff
@@ -533,6 +533,7 @@ foam.CLASS({
       // Don't save the 'save' command
       this.currentBlock.del();
 
+
```

Comment:
Why the extra line?

Reply:
if this is not important we can ignore it for now